<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Characteristic</class>
 <widget class="QWidget" name="Characteristic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_30" native="true">
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 8px;
background-color:rgb(248, 249, 251);
color: rgb(140, 139, 139);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_57">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="name_quality">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(248, 249, 251);
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string>Свойство</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="value_quality">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>22</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(248, 249, 251);
color: rgb(140, 139, 139);
outline: 0px;
padding-left: 10px;
margin-left: 20px;
margin-right: 5px;
border-top-left-radius: 0px;
border-bottom-left-radius: 00px;
border: none;

</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dimension_quality">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-right: 10px;
</string>
        </property>
        <property name="text">
         <string>°C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clean_quality">
        <property name="font">
         <font>
          <family>Montserrat</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color:rgb(138, 161, 233);
color:rgb(255, 255, 255);
font-weight: 600;
border-radius: 8px;
padding: 5px 15px;
outline: 0px;
}
QPushButton:hover{
background-color: rgb(125, 143, 202);
}</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
